src/posthog-core.ts

```diff
@@ -117,6 +117,7 @@ let ENQUEUE_REQUESTS = !SUPPORTS_REQUEST && userAgent?.indexOf('MSIE') === -1 &&
 
 export const defaultConfig = (): PostHogConfig => ({
     api_host: 'https://us.i.posthog.com',
+    flags_api_host: undefined,
```

Comment:
Not sure how I feel about `flags_api_host`. Decide is much more than flags.

Also more generally I don't see adblockers caring enough to only block the other domains and not this. We're entering a whack-a-mole situation for sure.

Also also will this even help?? The api_host is what is used to load the JS from the snippet which will be blocked. If the snippet doesn't load then there is no benefit to having a separate flags host as there is no code to call it anways 😓 

I realise I'm proposing problems without solutions. I'm just not super keen on this solution that adds more config for a user for potentially no real gains.

Reply:
The best solution I can think of is having the tracker be on a subpath of the same domain rather than a separate domain seeing as typically adblockers will allow the domain if it is the same as the webpage you are on.

Reply:
I included `static/array.js` in the filter, so that would go through the (hopefully) non-blocking domain as well. 

I don't think ad blockers would be looking specifically for array.js? I guessing what they look for is any URL that sends out lots of info when someone is on a page - that's our `/e/` and `/s/` paths. Those will all still go through the main `api_host` url, which is fine if an ad blocker blocks it (which is partially what an ad blocker is for anyway). 

`/decide` and `static/array.js` are NOT tracking endpoints so hopefully people aren't looking specifically for those and blocking them when found. If we name our endpoint something silly (`feature-flags-do-not-block.posthog.com`) then hopefully they will (correctly) ignore that domain traffic. I'm also down for putting this on a completely separate domain if we want to be more careful.